<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsLightsWidget</class>
 <widget class="QWidget" name="QgsLightsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cboLights"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnAddLight">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemoveLight">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="spinA0">
       <property name="maximum">
        <double>9.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="spinA1">
       <property name="maximum">
        <double>9.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="spinPositionX">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-9999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QDoubleSpinBox" name="spinPositionY">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-9999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="spinPositionZ">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-9999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Intensity</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="spinIntensity">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Attenuation (A&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;+A&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;*D+A&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;*D&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QgsColorButton" name="btnColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="spinA2">
       <property name="maximum">
        <double>9.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cboLights</tabstop>
  <tabstop>btnAddLight</tabstop>
  <tabstop>btnRemoveLight</tabstop>
  <tabstop>spinPositionX</tabstop>
  <tabstop>spinPositionY</tabstop>
  <tabstop>spinPositionZ</tabstop>
  <tabstop>btnColor</tabstop>
  <tabstop>spinIntensity</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
